stdlib/compression/_common/_streams.pyi

```diff
@@ -11,13 +11,24 @@ class _Reader(Protocol):
     def seekable(self) -> bool: ...
     def seek(self, n: int, /) -> Any: ...
 
+@type_check_only
+class _Decompressor(Protocol):
```

Comment:
Could we import this form `_compression` instead to avoid the duplication?

Reply:
As I understand it, these files completely duplicate each other, since `_compression`, according to [`VERSION`](https://github.com/python/typeshed/blob/0d05183d9b8ee97879b7364ed9a274069e298431/stdlib/VERSIONS#L31), is available up to 3.13